src/AspNetCore.Identity.MongoDB/AspNetCore.Identity.MongoDB.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Since this is a library project, this should target `netstandard` rather than `netcoreapp`. I made that change under my branch.